components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -228,6 +228,7 @@ tweaking their $case/namelist_scream.xml file.
       <Casename>${CASE}</Casename>
       <Output__Control>
         <Frequency>${REST_N}</Frequency>
+        <Frequency REST_OPTION="none">0</Frequency>
```

Comment:
We still operate under the convention that "last match takes precedence" right? So the line above will be overwritten by this one in case REST_OPTION is indeed none?

Reply:
Yep. I did some basic testing and the INSTANT.INVALID restart file was gone.